### PR TITLE
Added CDATA tag

### LINKED ISSUE #1772: No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)
**Summary:** 

I am seeing this in our logs:
```No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)```

It's probably the previously encountered issue about HTML tags present. Could you apply the usual fix?

### PR DESCRIPTION
**Pull request by Mansi449**
## Description (required)

Fixes #1772 No source text for Commons-android-strings-welcome_help_button_text (commons-android-strings)

Added the CDATA tag to the strings which had the `<u>` tag. Set the text of the corresponding textviews using the Html.fromHtml() function in the corresponding Java files.

## Tests performed (required)

Tested on device Motorola Moto G(5) (Android 7.0, API 24), with build variant betaDebug.

### CODE REVIEW

**Comment by Mansi449** [comment]
@maskaravivek @misaochan @Nikerabbit Could you please review my PR? Do tell me if there are any changes that I should make.

File: app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java [hunk 1]
```diff
@@ -66,6 +67,13 @@ public Object instantiateItem(ViewGroup container, int position) {
             }
             ViewHolder holder = new ViewHolder(layout);
             layout.setTag(holder);
+
+            if(position == PAGE_FINAL){
```
Comment (misaochan) [inline app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: Could you please explain why this was included in this PR? I don't see anything wrong with the code per se, but it seems to address a separate issue.

Comment (Mansi449) [reply app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: Sure @misaochan! Adding CDATA tag alone does not give the expected output. We have to use the setText(Html.fromHtml()) method. So, when the 'ViewGroup' object 'layout' holds the final layout page (which contains the string with CDATA tag as text), then only we have to set the text. That's why I have used an 'if' condition to know when 'layout' holds the targeted page.

Comment (misaochan) [reply app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: Make sense! Thanks. :) Happy to merge once this has been tested.

**Comment by neslihanturan** [comment]
Thanks @Mansi449 !

**Comment by Nikerabbit** [comment]
There's a missing `>` which I noticed when importing changes.

**Comment by neslihanturan** [comment]
Oh, @Mansi449 can you please add that one too?

**Comment by Mansi449** [comment]
@Nikerabbit @neslihanturan I'll fix that right away!

File: app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java (hunk 1 above)
Comment (domdomegg) [reply app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: Going back through `WelcomePagerAdapter` and didn't fully understand this - why is this enabled only for the beta version of the app?

**Pull request merged**